mm2src/coins/eth.rs

```diff
@@ -59,6 +59,10 @@ use web3::types::{Action as TraceAction, BlockId, BlockNumber, Bytes, CallReques
 use web3::{self, Web3};
 use web3_transport::{EthFeeHistoryNamespace, Web3Transport, Web3TransportNode};
 
+use crate::utxo::utxo_builder::UtxoConfError;
```

Comment:
`UtxoConfError` is a UTXO specific error enumeration. Please don't use it in `eth.rs`